dashboard/lib/services/lti.rb

```diff
@@ -179,13 +198,20 @@ def self.sync_course_roster(lti_integration:, lti_course:, nrps_sections:, secti
     end
 
     nrps_sections.keys.each do |lms_section_id|
-      section_name = nrps_sections[lms_section_id][:name]
+      nrps_section = nrps_sections[lms_section_id]
```

Comment:
Now that we're using the value as well as the key as we iterate, is there a better function to use than `keys.each`? Like some kind of object function where we can get both the key and value in the params of the iterator function.